lib/matplotlib/gridspec.py

```diff
@@ -141,7 +141,7 @@ def get_height_ratios(self):
         """
         return self._row_height_ratios
 
-    @_api.delete_parameter("3.7", "raw")
+   
     def get_grid_positions(self, fig, raw=False):
```

Comment:
`raw` must be deleted from the signature here, and all uses of `raw` should be removed in the function.